Cargo.toml

```diff
@@ -11,7 +11,7 @@ chrono = { version = "=0.4", features = ["clock"], default-features = false }
 displaydoc = "0.2"
 function_name = "0.3"
 loupe = "0.1"
-massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs.git", rev = "30b91d705b704287662b0e42457ece442005fa46" }
+massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs.git", rev = "84678fb77cf6d06d85d55e2c20502908ff292e61", features = ["tonic"] }
```

Comment:
Don't you need to target the `feature/chainid`branch of this repo for now (and change it to a commit id when merged to main)?

Reply:
massa-proto-rs has been automatically updated and the commit id is the last one (with the chain id modification).

BTW I've removed the features, there is no need for them here.